src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -217,7 +217,7 @@ private void readHeader() throws IOException {
         PositionalBufferedStream pbs = null;
         try {
             is = ParsingUtils.openInputStream(path);
-            if (isGZIPPath(path)) {
+            if (hasBlockCompressedExtension(path) || hasBlockCompressedExtension(new URI(path))) {
```

Comment:
it seems that the first part of if statement is only good for improving performance, but this isn't a performance bottleneck. is it needed for anything else?